docs/application/native/api/iot-headed/5.5/group__Evas__Object__Group.html

```diff
@@ -2491,7 +2518,7 @@
 <dl class="params"><dt><b>Parameters:</b></dt><dd>
   <table class="params">
     <tr><td class="paramdir">[in]</td><td class="paramname">x</td><td>Non-negative double value to use as horizontal weight hint. </td></tr>
-    <tr><td class="paramdir">[in]</td><td class="paramname">y</td><td>Non-negative double value to use as vertical weight hint. </td></tr>
+    <tr><td class="paramdir">[in]</td><td class="paramname">y</td><td>Non-negative double value to use as vertical weight hint.  2.3 </td></tr>
```

Comment:
![image](https://user-images.githubusercontent.com/9958374/102974360-bc57e600-4541-11eb-838d-c648e4b3a7ef.png)

Same issue to line 2475.

Reply:
checked

Reply:
https://review.tizen.org/gerrit/#/c/platform/upstream/efl/+/250985/